client/packages/glsp-sprotty/src/features/execute/execute-command.ts

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2018 EclipseSource Services GmbH.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *   
+ * Contributors:
+ * 	Tobias Ortmayr - initial API and implementation
+ ******************************************************************************/
+import { Action, MouseListener, SModelElement, findParentByFeature } from "sprotty/lib";
+import { isCommandExecutor } from "./model";
+
+export class ExecuteServerCommandAction implements Action {
+    static readonly KIND = "executeServerCommand"
+    kind = ExecuteServerCommandAction.KIND;
```

Comment:
Wouldn't it be good if the command had an optional list of element ids to provide a context for the command execution?

Reply:
You are right. We should probably use a generic options map (<String,String>) though for passing additional arguments/context information etc.. Similar to how it'S handled in the RequestModelAction.